api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/NextPaymentCycleSummary.java

```diff
@@ -14,11 +14,16 @@
 
     private int numberOfChildrenTurningOne;
     private int numberOfChildrenTurningFour;
+    private boolean childrenUnderFourPresentAtEndOfCycle;
```

Comment:
It feels to me like we'll also need to store whether they are considered pregnant for the next cycle too otherwise the calculations will need to be made elsewhere when they could nicely be encapsulated here.

Reply:
I hadn't wanted to include pregnancy here as the summary object has only referenced children until now.

Reply:
Adeel and I will look at it in the next PR and see where pregnancy fits best